entry.go

```diff
@@ -260,6 +261,13 @@ func (entry *Entry) log(level Level, msg string) {
 	}
 }
 
+func (entry *Entry) getBufferPool() (pool BufferPool) {
+	if entry.Logger.BufferPool != nil {
```

Comment:
Can we have that protected by the logger mutex ?

Reply:
The only place to call this method is already protected by a mutex.

Reply:
ok, I missed that 👍